### PR TITLE
Added support BMP280 Sensor

### PR DESCRIPTION
**Pull request by Jovian-Dsouza**
Added Support for BMP280 I2C Barometer. These are cheap and provide a higher resolution than BMP085  Barometers.

### CODE REVIEW

File: Sensors.cpp [hunk 1]
```diff
@@ -471,6 +471,154 @@ uint8_t Baro_update() {                   // first UT conversion is started in i
...
+ *  
+ *  baroTemperature unit is 0.01 deg C 
+ *  baroPressure unit is Pa  
+ *   
+ */
+ 
+void bmp280_measure(void)
+{
+  uint8_t data[BMP280_RAWDATA_BYTES];
+  int32_t temp_raw, pres_raw, var1, var2, t_fine;
```
Comment (fakcior) [inline Sensors.cpp]: Variables var1 and var2 must be int64 type.

**Review by fakcior — CHANGES_REQUESTED** [review]

**Review by Aashray446 — APPROVED** [review]

**Comment by haxord7hasib** [comment]
could not find anything about bmp280 on the config. h file !!

File: Sensors.cpp (hunk 1 above)
Comment (haxord7hasib) [reply Sensors.cpp]: I could not find anything can you please help

**Comment by Jovian-Dsouza** [comment]
> could not find anything about bmp280 on the config. h file !!

Here, 
https://github.com/multiwii/multiwii-firmware/commit/d541d5a4ca42b28386bdcb270769cff846826c47

The commit was not merged

**Comment by Jovian-Dsouza** [comment]
> could not find anything about bmp280 on the config. h file !!

Here checkout this new PR #31

**Pull request closed without merging**